admin/checkservices

```diff
@@ -298,7 +298,7 @@ main() {
     echo "Found: ${#broken_services[@]}"
     if (( ${#broken_services[@]} )); then
         display_restart "${broken_services[@]}"
-        if confirm 'Execute?'; then
+        if (( $RESTART )) && confirm 'Execute?'; then
```

Comment:
Testing this out, I think this condition should be moved up as now the output is kinda confusing:

```
[jelle@t14s][~/projects/contrib]%sudo ./admin/checkservices -R -P
:: Reload systemd
:: Services with broken maps files
Error:: Unable to parse pid file for systemd-udevd.service.
Found: 6
-------8<-------------------------------8<---------
systemctl restart 'atop.service'
systemctl restart 'cups.service'
systemctl restart 'systemd-journald.service'
systemctl restart 'systemd-logind.service'
systemctl restart 'systemd-machined.service'
systemctl restart 'systemd-timesyncd.service'
-------8<-------------------------------8<---------
:: Services missing on the system bus
```

Would be better to merge the `$RESTART` with the `if` above.

Reply:
@uggedal Are you able to take a look at the above? I just opened a round of PRs with some fixes/improvements for `checkservices`. It would be nice if we could merge this one with them :)

Reply:
I agree the output is kinda confusing that way, but I think getting the list of broken services with the `-R` option would be great (to act as a sort of dry-run).

Reply:
I propose the following: 

```
diff --git a/admin/checkservices b/admin/checkservices
index f2a59b4..e0518d7 100755
--- a/admin/checkservices
+++ b/admin/checkservices
@@ -297,22 +297,30 @@ main() {
     local -a broken_services=($(get_broken_maps))
     echo "Found: ${#broken_services[@]}"
     if (( ${#broken_services[@]} )); then
-        display_restart "${broken_services[@]}"
-        if confirm 'Execute?'; then
-            arrow 'Restart broken services'
-            restart_services "${broken_services[@]}"
-        fi
+       if (( RESTART )); then
+            display_restart "${broken_services[@]}"
+            if confirm 'Execute?'; then
+                arrow 'Restart broken services'
+                restart_services "${broken_services[@]}"
+            fi
+       else
+           display_restart "${broken_services[@]}" | sed "s/systemctl restart //g"
+       fi
     fi

     arrow 'Services missing on the system bus'
     local -a missing_services=($(get_missing_dbus))
     echo "Found: ${#missing_services[@]}"
     if (( ${#missing_services[@]} )); then
-        display_restart "${missing_services[@]}"
-        if confirm 'Execute?'; then
-            arrow 'Restart missing services'
-            restart_services "${missing_services[@]}"
-        fi
+        if (( RESTART )); then
+           display_restart "${missing_services[@]}"
+            if confirm 'Execute?'; then
+                arrow 'Restart missing services'
+                restart_services "${missing_services[@]}"
+            fi
+       else
+           display_restart "${missing_services[@]}" | sed "s/systemctl restart //g"
+       fi
     fi

     # list only failed systemd units
```

Which results in the following output:

```
 antiz@Arch-Desktop  .../contrib/admin   master ●  sudo ./checkservices -R
:: Run pacdiff
:: Reload systemd
:: Services with broken maps files
Error:: Unable to parse pid file for getty@tty1.service.
Error:: Unable to parse pid file for systemd-udevd.service.
Found: 12
-------8<-------------------------------8<---------
'bluetooth.service'
'dbus-broker.service'
'docker.service'
'firewalld.service'
'NetworkManager.service'
'polkit.service'
'rtkit-daemon.service'
'systemd-journald.service'
'systemd-logind.service'
'systemd-timesyncd.service'
'systemd-userdbd.service'
'udisks2.service'
-------8<-------------------------------8<---------
:: Services missing on the system bus
Found: 0
```